cmd/limactl/unprotect.go

```diff
@@ -0,0 +1,46 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/lima-vm/lima/pkg/store"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+func newUnprotectCommand() *cobra.Command {
+	var unprotectCommand = &cobra.Command{
+		Use:               "unprotect INSTANCE [INSTANCE, ...]",
+		Short:             "Unprotect an instance",
+		Args:              WrapArgsError(cobra.MinimumNArgs(1)),
+		RunE:              unprotectAction,
+		ValidArgsFunction: unprotectBashComplete,
+	}
+	return unprotectCommand
+}
+
+func unprotectAction(_ *cobra.Command, args []string) error {
+	var errs []error
+	for _, instName := range args {
+		inst, err := store.Inspect(instName)
+		if err != nil {
+			errs = append(errs, fmt.Errorf("failed to inspect instance %q: %w", instName, err))
+			continue
+		}
+		if !inst.Protected {
+			logrus.Warn("Instance %q isn't protected", instName)
+			// try Unprotect() anyway; this may help working around pontential bugs
```

Comment:
@jandubois FYI

Reply:
I don't think this is necessary. And I find it confusing to see both messages:

```
Instance "foo" isn't protected
Unprotected "foo"
```

Reply:
Fixed